### ISSUE TITLE
species as query parameter?

### ISSUE THREAD

**Issue by edeutsch** [HUPO-PSI/proxi-schemas#66]
Also in my implementation notes was the idea that "species" should be an input parameter to all of the endpoints. One can imagine wanting to constrain any of those queries to limit results to just one species.

What do you think?

**Comment by javizca** [comment]
Yes, I think they could be input parameters in all cases. I give preference to the TaxID which is more unique that the species name.

**Comment by ypriverol** [comment]
No easy way to implement that in PRIDE @javizca. I guess we really need to see how resources can implement that.
